### PR TITLE
[Storage] Fix help file format per static analysis result

### PR DESCRIPTION
**Pull request by yifanz0**
## Description
Fix help file formatting according to the static analysis failure - 
https://dev.azure.com/azclitools/public/_build/results?buildId=28893&view=logs&j=b9afb22f-0837-5d02-bc24-6c30aee08bcf&t=990f6f37-4a21-5927-092a-220f38a460e9

## Checklist

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

**Review by blueww — CHANGES_REQUESTED** [review]
For the example without output, suggest remove "PS C:\>" instead of adding Skip tag.
If the example has single output, please separate the input and output.
Else If the example has multiple output, we should  use the skip tag.

**Comment by BethanyZhou** [comment]
@yifanz0 , there are still some issues reported by our static analyzer. Please correct them.

**Review by BethanyZhou — APPROVED** [review]

**Comment by BethanyZhou** [comment]
@blueww , do you have any comments? If everything looks good to me, I will merge this PR.

File: src/Storage/Storage.Management/help/New-AzStorageContext.md [hunk 1]
```diff
@@ -180,39 +180,39 @@ The second command creates a context for the account named ContosoGeneral that u
...
+```powershell
+$SasToken = New-AzStorageContainerSASToken -Name "MyContainer" -Permission "rad"
+New-AzStorageContext -SasToken $SasToken -BlobEndpoint "https://myaccountname.blob.core.windows.net/" -TableEndpoint "https://myaccountname.table.core.windows.net/" -FileEndpoint "https://myaccountname.file.core.windows.net/" -QueueEndpoint "https://myaccountname.queue.core.windows.net/"
 ```
 
 The first command generates an SAS token by using the New-AzStorageContainerSASToken cmdlet for the container named MyContainer, and then stores that token in the $SasToken variable.
-The second command creates a context for the account named myaccountname that uses the SAS token and a specified blob endpoint, table endpoint, file endpoint, and queue endpoint. 
+The second command creates a context that uses the SAS token and a specified blob endpoint, table endpoint, file endpoint, and queue endpoint. 
 
 ### Example 14: Create ea context by using the OAuth Authentication with a specified blob endpoint
```
Comment (blueww) [inline src/Storage/Storage.Management/help/New-AzStorageContext.md:213]: ea - > a

**Review by blueww — APPROVED** [review]
LGTM expect small typo need fix.

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**